Healthy/Classes/Modules/Login/LoginViewController.swift

```diff
@@ -101,50 +105,68 @@ private extension LoginViewController {
     func configureViewModelInputs() {
         bindTextFieldsChanges()
     }
-    
+
     func configureViewModelOutputs() {
         bindLoadingIndicator()
         bindErrorMessage()
         bindButtonState()
         bindLoginStatus()
     }
-    
+
     func bindLoadingIndicator() {
-        viewModel.isLoadingIndicatorPublisher
-            .sink { [weak self] isLoading in
-                guard let _ = self else { return }
-                
-                // TODO: Show loading indicator.
-            }
-            .store(in: &subscriptions)
+//        viewModel.isLoadingIndicatorPublisher
+//            .sink { [weak self] _ in
+//                guard let _ = self else { return }
+//
+//                // TODO: Show loading indicator.
+//            }
+//            .store(in: &subscriptions)
```

Comment:
remove commented code 
OR 
Add your loading indicator view 
Check this link that will help you make a simple, beautiful and reusable indicator view component 
https://youtu.be/iHeeOULNdFA